src/OpenLoco/src/World/CompanyManager.cpp

```diff
@@ -413,6 +414,7 @@ namespace OpenLoco::CompanyManager
             if (metric < bestInstalledValue)
             {
                 bestInstalled = std::vector<ObjectHeader>{ *installed.second._header };
+                bestInstalledValue = metric;
```

Comment:
Does this warrant a changelog entry?

Reply:
What do you mean

Reply:
Oh, the implication in the commit message was that the value was wrong on master, no?

Reply:
oh right that was an implementation mistake.

Reply:
Ahh, I see now that code in question was introduced in the first commit. Never mind, then!